### PR TITLE
Feature/jenkins

### PR DESCRIPTION
**Pull request by cmccully**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    volumes:
+    - nrese2e:/archive
+    labels:
+      io.rancher.container.pull_image: always
+      io.rancher.container.start_once: 'true'
+  NRESPipelineTest:
+    image: ${DOCKER_IMG}
+    environment:
+      BOSUN_HOSTNAME: alertsdev.lco.gtn
+      BROKER_PASSWORD: guest
```
Comment (kurczynski) [inline docker-compose.yml]: Credentials should be in the Jenkins credentials store.

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      BROKER_URL: amqp://guest:guest@nresceleryrabbitmq:5672/
+      BROKER_USERNAME: guest
```
Comment (kurczynski) [inline docker-compose.yml]: Credentials should be in the Jenkins credentials store.

Comment (cmccully) [reply docker-compose.yml]: Done.

File: docker-compose.yml (hunk 1 above)
Comment (cmccully) [reply docker-compose.yml]: Done.

**Pull request merged**